Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -76,6 +89,44 @@ private Task AddProjectsToSolution()
                 new[] { projectInfo, projectTestInfo });
         }
 
+        private Task AddInternalNugetServerToNugetConfig()
+        {
+            var nugetConfigFileName = "NuGet.config";
+            var nugetConfigFilePath = Path.Combine(SolutionDirectory, nugetConfigFileName);
+
+            var nugetConfigModification = CreateInternalNugetServerModification();
+            return _nugetConfigurationManipulator.ModifyNugetConfiguration(nugetConfigFilePath, nugetConfigModification);
+        }
+
+        private NugetConfigurationModification CreateInternalNugetServerModification()
+            => new NugetConfigurationModificationBuilder(_nugetPackageSourceManipulator)
+                .AddPackageSource(CreateInternalNugetServer())
+                .Build();
+
+        private NugetPackageSource CreateInternalNugetServer()
+            => new NugetPackageSource("Internal Nuget Server", "https://nuget.messerli.ch/v3/index.json");
+
+        private Task AddMsBuildSdkToGlobalJson()
+        {
+            var globalJsonFileName = "global.json";
```

Comment:
Make this a constant

Reply:
Addressed in: [Make variable const](https://github.com/messerli-informatik-ag/meta-generator/pull/22/commits/8c5bdf4b730639832151dbbe7f46567e33bc5d4a)